.circleci/src/jobs/notify_release.yml

```diff
@@ -0,0 +1,25 @@
+# Create a release (with release notes) on github and send a mattermost notification.
+parameters:
+  production_delivery:
+    description: Whether builds are delivered to the production or beta lane of the play store.
+    type: boolean
+docker:
+  - image: cimg/node:19.1.0
+resource_class: small
+steps:
+  - prepare_workspace
+  - restore_environment_variables
+  - install_app_toolbelt
+  - run:
+      name: Create github release
+      command: echo "export RELEASE_ID='$(app-toolbelt v0 release create all ${NEW_VERSION_NAME} ${NEW_VERSION_CODE} --deliverino-private-key ${DELIVERINO_PRIVATE_KEY} --owner ${CIRCLE_PROJECT_USERNAME} --repo ${CIRCLE_PROJECT_REPONAME} --production-release << parameters.production_delivery >> --should-use-predefined-release-notes false)'" >> ${BASH_ENV}
+  - run:
+      name: Upload android apks to github release
+      command: app-toolbelt v0 release upload --releaseId ${RELEASE_ID} --files "$(ls ~/attached_workspace/*.apk)" --deliverino-private-key ${DELIVERINO_PRIVATE_KEY} --owner ${CIRCLE_PROJECT_USERNAME} --repo ${CIRCLE_PROJECT_REPONAME}
+  - run:
+      name: Upload ios ipas to github release
+      command: app-toolbelt v0 github-release-asset upload --releaseId ${RELEASE_ID} --files "$(ls ~/attached_workspace/*.ipa)" --deliverino-private-key ${DELIVERINO_PRIVATE_KEY} --owner ${CIRCLE_PROJECT_USERNAME} --repo ${CIRCLE_PROJECT_REPONAME}
+  - notify:
+      success_message: <<^ parameters.production_delivery >>[Beta] <</ parameters.production_delivery >>Ehrenamtskarte Bayern und Sozialpass Nürnberg ${NEW_VERSION_NAME} have been released successfully! https://github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/releases/tag/${NEW_VERSION_NAME}-all
+      channel: releases
+      allow-all-branches: true
```

Comment:
This should be set to false, as we only want to allow "release-*" and "main" branches.